.github/workflows/oracle-workflow-test.yml

```diff
@@ -53,12 +54,13 @@ jobs:
         run: |
           if [[ $QUAY_IO_USERNAME == '' ]] && [[ $QUAY_IO_PASSWORD == '' ]]; then
             echo "HAS_QUAY_IO=false" >> $GITHUB_OUTPUT
+            exit -1
```

Comment:
This will cause any Oracle-related PR to always be in a failed state and will cancel any other pending action workflows that are running per the GH Action docs.  Can we not simply leave this as it is?

Reply:
I added a commit to revert this change.

Reply:
@Naros I don't like that "solution". I think we then could remove that check at all because now it will always passes, right?
I suggest we either change the test code to work on PRs too or at least we change the Oracle CI test to not run the entire test suite but only the compile check on PRs and add another, new workflow for tests on the main branch.
Maybe we could even do: run the proper check here. Then, depending on the result we run either a compile-only test or the whole test suite when secrets are available. That way we can have PRs on the main repo here like mine that will work and we can test the whole suite. I would personally feel much more comfortable if I do changes to such a black box like Oracle DB connector.
If you and @jpechane like on of the options I can implement one of those solutions.

Reply:
I filed https://issues.redhat.com/browse/DBZ-7103 for this.